generics/constants/endpoints.js

```diff
@@ -47,5 +47,7 @@ module.exports = {
     FILES_DOWNLOADABLE_URL: "/v1/cloud-services/files/getDownloadableUrl",
     OBSERVATION_DETAILS : "/v1/observations/details",
     USER_READ_V5 : "/v5/user/read",
-    GET_LOCATION_DATA : "/v1/location/search"
+    GET_LOCATION_DATA : "/v1/location/search",
+    CERTIFICATE_CREATE : "/api/v1/ProjectCertificate",
+    CERTIFICATE_API_CALLBACK : "/api/userProject/mlproject/v1/certificateCallback"
```

Comment:
@VISHNUDAS-tunerlabs why we need to add callback url here?

Reply:
@priyanka-TL this end point is used in cetificate service-